directus-extension-charging-stations-bundle/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+	"compilerOptions": {
```

Comment:
I think this is very nitpicky: Should our tsconfig be consistent of what is in the other packages? Or does directus need some special things?

Reply:
this is automatically generated when you use the command `npx create-directus-extension@latest` to create an empty directus extension package of your choice of extension type. I doubt it *needs* any of this though. I'll make it match our other tsconfigs

Reply:
Updated!